wordpress-mobile/WordPress-iOS
# The blogging reminders configuration is refreshed in Site Settings.

The blogging reminders configuration is now refreshed in Site Settings.

## To test:

Using a blog that has no blogging reminders configuration.

1. Go to My Site > Site Settings
2. Check that the blogging reminders cell shows "None set"
3. Enable it for 1 day and complete the flow.
4. Make sure the cell now reads "Once a week"
5. Change to any other number less than 7 and make sure it's refreshed accordingly.
6. Change to all 7 days and check that it reads "Every day".

## Regression Notes

1. Potential unintended areas of impact

None.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
